### PR TITLE
Update adding-windows-nodes.md

### PR DESCRIPTION
**Pull request by guzj11**
sync with the english version with same format.

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md [hunk 1]
```diff
@@ -269,14 +269,18 @@ to do so are available at [Install Docker Engine - Enterprise on Windows Servers
...
+{{< note >}}
 你必须安装 `Containers` 功能特性并安装 Docker 工具。相关的指令可以在
 [Install Docker Engine - Enterprise on Windows Servers](https://docs.mirantis.com/docker-enterprise/v3.1/dockeree-products/docker-engine-enterprise/dee-windows.html)
 处找到。
+{{< /note >}}
 
+{{< note >}}
 Windows 节的所有代码片段都需要在 PowerShell 环境中执行，并且要求在
 Windows 工作节点上具有提升的权限（Administrator）。
+{{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:283]: 这些 note 短代码是有意删除的。完全必要到处使用 note。

Comment (guzj11) [reply content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:283]: get it, i will remove the note labels, only update the spelling

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md:283]: Sorry, this case is not common. But we do have the freedom to tail the content to our need.

**Comment by guzj11** [comment]
> /approve

can this commit be merged now?

File: content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md (hunk 1 above)
Comment (guzj11) [reply content/zh/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md]: maybe this commit can be merged?

**Review by markthink — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**